--- venv/lib/python3.12/site-packages/gymnasium/envs/mujoco/mujoco_rendering.py	2024-08-08 14:46:34.981953776 +0100
+++ patches/mujoco_rendering.py	2024-08-08 14:49:43.668876525 +0100
@@ -590,7 +590,7 @@
 
         self.add_overlay(bottomleft, "FPS", "%d%s" % (1 / self._time_per_render, ""))
         self.add_overlay(
-            bottomleft, "Solver iterations", str(self.data.solver_iter + 1)
+            bottomleft, "Solver iterations", str(self.data.solver_niter + 1)
         )
         self.add_overlay(
             bottomleft, "Step", str(round(self.data.time / self.model.opt.timestep))
